controllers/controller.js

```diff
@@ -12,6 +12,15 @@ exports.getApi = (request, response) => {
         })
 };
 
+exports.getTopics = (request, response) => {
+    fetchTopics().then(({ rows }) => {
+        response.send({ rows });
+    })
```

Comment:
Should be sending back on the key of `topics` not `rows`

Reply:
Updated, thanks.